source/user-manual/agents/query-configuration.rst

```diff
@@ -39,7 +39,7 @@ It also has the capability to show the configuration in JSON or XML formats for
   :align: center
   :width: 100%
 
-Now for example, if you want to know the current cluster configuration from one of your *Wazuh managers*, click on the *Management > Configuration* tab, select the desired node and then click on the *Cluster* section:
+Now for example, if you want to know the current cluster configuration from one of your *Wazuh managers*, click on the **Settings** module, select the desired node and then click on the **Cluster** section:
```

Comment:
Do we mean `Server management > Settings`?

Reply:
I will add this.